compiler/next/include/chpl/uast/all-uast.h

```diff
@@ -26,6 +26,7 @@
 #include "chpl/uast/BytesLiteral.h"
 #include "chpl/uast/CStringLiteral.h"
 #include "chpl/uast/Call.h"
+#include "chpl/uast/Class.h"
```

Comment:
I don't think it needs to be, because it'll be included by Class etc, but also because this header exists so the concrete classes can be instantiated. I wouldn't object to adding it but there are other generic parents not in here (e.g. TypeDecl).